pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,448 @@
+    echo "${OPTION}=NONE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-timestamps" ]; then
+    echoit "  > Adding possible values SYSTEM, UTC for option '${OPTION}' to the final list..."
+    echo "${OPTION}=UTC" >> ${OUTPUT_FILE}
+    echo "${OPTION}=SYSTEM" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--master-info-repository" ]; then
+    echoit "  > Adding possible values FILE, TABLE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--relay-log-info-repository" ]; then
+    echoit "  > Adding possible values FILE, TABLE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=FILE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TABLE" >> ${OUTPUT_FILE}
+    elif [ "${OPTION}" == "--windowing-use-high-precision" ]; then
+    echoit "  > Adding possible values 0, 1 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--default-storage-engine" ]; then
+    echoit "  > Adding possible value InnoDB for option '${OPTION}' to the final list..."
+    echo "${OPTION}=InnoDB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${VALUE}" == "TRUE" -o "${VALUE}" == "FALSE" -o "${VALUE}" == "ON" -o "${VALUE}" == "OFF" -o "${VALUE}" == "YES" -o "${VALUE}" == "NO" ]; then
+    echoit "  > Adding possible values TRUE/ON/YES/1 and FALSE/OFF/NO/0 (as a universal 1 and 0) for option '${OPTION}' to the final list..."
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+  elif [[ "$(echo ${VALUE} | tr -d ' ')" =~ ^-?[0-9]+$ ]]; then
+    if [ "${VALUE}" != "0" ]; then 
+      echoit "  > Adding int values (${VALUE}, -1, 0, 1, 2, 12, 24, 254, 1023, 2047, -1125899906842624, 1125899906842624) for option '${OPTION}' to the final list..."
+      echo "${OPTION}=${VALUE}" >> ${OUTPUT_FILE}
+    else
+      echoit "  > Adding int values (-1, 0, 1, 2, 12, 24, 254, 1023, 2047, -1125899906842624, 1125899906842624) for option '${OPTION}' to the final list..."
+    fi
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=12" >> ${OUTPUT_FILE}
+    echo "${OPTION}=24" >> ${OUTPUT_FILE}
+    echo "${OPTION}=254" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1023" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2047" >> ${OUTPUT_FILE}
+    echo "${OPTION}=-1125899906842624" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1125899906842624" >> ${OUTPUT_FILE}
+  elif [ "${VALUE}" == "" -o "${VALUE}" == "(No" ]; then
+    echoit "  > Assert: Option '${OPTION}' is blank by default and not programmed into the script yet, please cover this in the script..."
+    exit 1
+  else
+    echoit "  > ${OPTION} IS NOT COVERED YET, PLEASE ADD!!!"
+    #exit 1
```

Comment:
Why is this commented? Shouldn't this be uncommented, so that the script tells us about missing variables